templates/etc/env.php.j2

```diff
@@ -130,6 +130,32 @@ return array (
   {% if magento_run_message_queue_consumers_separately %}
   'cron_consumers_runner' => array(
       'cron_run' => false
-  )
+  ),
+  {% endif %}
+  {% if magento_queue_amqp is defined%}
```

Comment:
here seem to be a mistake, you duplicated the `magento_queue_amqp`  condition, which has very little to do with SSO/SAP credentials